src/traces/surface/attributes.js

```diff
@@ -113,9 +113,18 @@ var attrs = module.exports = overrideAll({
     },
 
     text: {
-        valType: 'data_array',
-        description: 'Sets the text elements associated with each z value.'
+        valType: 'string',
+        role: 'info',
+        dflt: '',
+        arrayOk: true,
+        editType: 'calc',
```

Comment:
Oh this is in an `overrideAll` so you don't need `editType` - which I think applies to a few more of the additions in this PR too.

Reply:
(I see you 🔪 some `editType` lines, just not the one I made the comment directly on 😅 )

Reply:
Oops. Fixed in -> 13a3ecf